fog/view/connection/Cargo.toml

```diff
@@ -23,5 +24,7 @@ mc-fog-uri = { path = "../../uri" }
 mc-fog-view-protocol = { path = "../protocol" }
 
 # third-party
+futures = "0.3"
 grpcio = "0.10.2"
 retry = "1.3"
+tokio = "1.19.2"
```

Comment:
Does this work? I expect we need to add `features = ["rt-multi-thread"]`

Reply:
Added the full tokio features. Now it works.